qutebrowser/mainwindow/tabwidget.py

```diff
@@ -151,7 +151,7 @@ def update_tab_title(self, idx, field=None):
 
         fields = self.get_tab_fields(idx)
         fields['current_title'] = fields['current_title'].replace('&', '&&')
-        fields['index'] = idx + 1
+        fields['index'] = str(idx + 1).rjust(2)
```

Comment:
I don't think this will actually align the indexes, if you have > 100 tabs, the tabs over 100 will be misaligned. In addition with less than 10 tabs, there will be a pointless space.

Reply:
yep, both are known and were mentioned in the PR description to some extend 🙂 But on the other hand it's a simple patch that fixes a problem, running more than 99 tabs is pretty much unrealistic today in qutebrowser as it keeps all of them in memory, and an extra space doesn't seem to be that bad to me in either horizontal or vertical setup. 

but what do you think? do you have an alternative suggestion for how to solve the alignment? 🙂

Reply:
I think it shouldn't be hardcoded but based on the number of tabs:
```python
fields['index'] = str(idx + 1).rjust(len(str(self.count())))
```
And either apply it only for vertical tab bar or create a new placeholder (@Jan-PieterBaert's suggestion below). Over 99 tabs is not unrealistic, many people sometimes have that.

Reply:
I like both ideas! 🙂